pkg/antctl/raw/multicluster/get/member_token.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package get
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+	"antrea.io/antrea/pkg/antctl/transform/member_token"
+)
+
+type tokenOptions struct {
+	namespace     string
+	outputFormat  string
+	allNamespaces bool
+	k8sClient     client.Client
+}
+
+var cmdToken *cobra.Command
+
+var optionsToken *tokenOptions
+
+var tokenExamples = strings.Trim(`
+# Get all Member Tokens in default Namesapce
```

Comment:
You used both "Member Token" and "member token". Member token is not a resource, so I would suggest to use "member token".